### DISCUSSION TITLE
Cohort Component

### DISCUSSION THREAD

**Discussion by mcrascal** [evidence-dev/evidence#2364] [General]
**Feature Description**
Standalone support for a nice cohort table.

Should accept three columns: 
1. Cohort (date)  
2. Age (number) 
3. Metric (e.g. NRR, any number, respecting formatting tags) 

Additional affordances: 
* Freeze cohort rows and age columns on horizontal or vertical scroll within the component 
* Treatment of "empty" cohort + time period combinations (e.g. completely blank) 
* Colour gradient based on the range of values for the metric, potentially with an option for whether higher or lower is better (see big value) 

Future enhancements: 
* Accept a cohort metric column (e.g. size of cohort) which is at the grain of the cohorts, and present that alongside the cohort.

**Comment by spatialy** [comment]
This will be nice to have
